### PR TITLE
[Doc] Update Meta Recovery

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Signed-off-by: EsoragotoSpirit <wanglichen@starrocks.com>

## What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.3
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

**Review by wangsimo0 — DISMISSED** [review]

File: docs/zh/administration/Meta_recovery.md [hunk 1]
```diff
@@ -664,6 +665,42 @@ jstat -gcutil pid 1000 1000
...
+
+4. 查看当前节点是否为 Leader 节点。
+
+   ```SQL
+   SHOW FRONTENDS;
+   ```
+
+   如果字段 `Role` 为 `LEADER`，说明该 FE 节点为 Leader FE 节点。确保返回的是当前 FE 节点的 IP 地址。
+
+5. 如果数据和元数据完整，且该节点的角色是 Leader，可以删除之前添加的配置并重新启动节点。
```
Comment (gengjun-git) [inline docs/zh/administration/Meta_recovery.md:695]: `可以删除之前添加的配置并重新启动节点。`
这里改成 `需要删除bdbje_reset_election_group = true配置并重启`吧，这个配置是一定要删掉的

File: docs/zh/administration/Meta_recovery.md [hunk 2]
```diff
@@ -664,6 +665,42 @@ jstat -gcutil pid 1000 1000
...
+
+   ```Properties
+   bdbje_reset_election_group = true
+   ````
+
+3. 启动该 FE 节点。
+
+   ```Bash
+   # 将 <fe_ip> 替换为新 FE 节点的 IP 地址（priority_networks），
+   # 并将 <fe_edit_log_port>（默认：9010）替换为新 FE 节点的 edit_log_port。
```
Comment (gengjun-git) [inline docs/zh/administration/Meta_recovery.md:683]: # 并将 <fe_edit_log_port>（默认：9010）替换为新 FE 节点的 edit_log_port。
这一句好像没有必要，默认9010就行。

File: docs/zh/administration/Meta_recovery.md [hunk 3]
```diff
@@ -664,6 +665,42 @@ jstat -gcutil pid 1000 1000
...
+4. 查看当前节点是否为 Leader 节点。
+
+   ```SQL
+   SHOW FRONTENDS;
+   ```
+
+   如果字段 `Role` 为 `LEADER`，说明该 FE 节点为 Leader FE 节点。确保返回的是当前 FE 节点的 IP 地址。
+
+5. 如果数据和元数据完整，且该节点的角色是 Leader，可以删除之前添加的配置并重新启动节点。
+6. 现在，您已成功通过元数据备份启动了新的 Leader FE 节点。您可以使用新 Leader FE 节点作为 Helper 启动 Follower 节点。
```
Comment (gengjun-git) [inline docs/zh/administration/Meta_recovery.md:696]: `启动` 改为`添加`比较合适

File: docs/zh/administration/Meta_recovery.md (hunk 2 above)
Comment (EsoragotoSpirit) [reply docs/zh/administration/Meta_recovery.md:683]: 一般文档不默认用户会使用默认值，写清楚就可以了

**Review by gengjun-git — APPROVED** [review]

**Review by yingtingdong — APPROVED** [review]

**Pull request merged**